networkit/test/test_scd.py

```diff
@@ -46,5 +46,41 @@ def testSCD(self):
 
 			self.assertLessEqual(cond, 0.5, "{} has too large conductance".format(name))
 
+	def testSCDGroundTruth(self):
+		G = nk.Graph(5, directed=True)
+		G.addEdge(0,1)
+		G.addEdge(2,1)
+		G.addEdge(1,3)
+		G.addEdge(0,4)
+		G.addEdge(3,2)
+
+		ground = nk.Cover(5)
+		ground.allToSingletons()
+		ground.addToSubset(1,2)
+		ground.addToSubset(2,3)
+
+		gt = nk.scd.SCDGroundTruthComparison(G, ground, {1:[1,2]}, True)
+		gt.run()
+		self.assertDictEqual(gt.getIndividualJaccard(), {1: 0.5})
+		self.assertDictEqual(gt.getIndividualPrecision(), {1: 0.5})
+		self.assertDictEqual(gt.getIndividualRecall(), {1: 1.0})
+		self.assertDictEqual(gt.getIndividualF1(), {1: 0.6666666666666666})
+		self.assertEqual(gt.getAveragePrecision(), 0.5)
+		self.assertEqual(gt.getAverageRecall(), 1.0)
+		self.assertAlmostEqual(gt.getAverageF1(), 0.667, 3)
+		self.assertEqual(gt.getAverageJaccard(), 0.5)
+
+	def testExpandOneCommunity(self):
+		nk.setSeed(42, False)
+		ground = nk.Cover(self.G.upperNodeIdBound())
+		ground.setUpperBound(1)
+		for i in range(10):
+			ground.addToSubset(0, i)
+		
+		bfs = nk.scd.RandomBFS(self.G, ground)
```

Comment:
Better test `RandomBFS` itself as part of `testSCD` and test `expandOneCommunity` (especially the TypeError exception) on its own

Reply:
the type error exception is added, the problem with including randomBFS into the existing testSCD is that the getConductance() value does not fit in the current assertions